app/admin/travels.rb

```diff
@@ -1,5 +1,5 @@
 ActiveAdmin.register Travel do
-  menu parent: 'Applicant Info', priority: 3
+  # menu parent: 'Applicant Info', priority: 3
```

Comment:
The travel feature should not be removed since we have travel records from past camps. This entry will not be used going forward but will allow for historic data viewing. I would not comment this line

Reply:
Ok, let it be there. I returned the menu link to Travels.